subsys/bluetooth/adv_prov/providers/Kconfig.fast_pair

```diff
@@ -67,4 +67,13 @@ config BT_ADV_PROV_FAST_PAIR_AUTO_SET_PAIRING_MODE
 	  Pair pairing mode and advertised data if generated advertising data
 	  is instantly used in advertising packets.
 
+config BT_ADV_PROV_FAST_PAIR_STOP_DISCOVERABLE_ON_RPA_ROTATION
```

Comment:
Please denote in the commit message that Fast Pair advertising data provider is updated too